plugins/dynamodb/plugin.spec.yaml

```
@@ -104,6 +104,58 @@ connection:
     example: us-east-1
     default: us-east-1
 actions:
+  get_item:
+    title: GetItem
+    description: The GetItem operation returns a set of attributes for the item with the given primary key. If there is no matching item, GetItem does not return any data and there will be no Item element in the response 
```

I think we should keep it to one sentence and not include any full stops.
https://docs.rapid7.com/insightconnect/style-guide#punctuation
```suggestion
    description: The GetItem operation returns a set of attributes for the item with the given primary key
```